### ISSUE TITLE
Docker image for ossec-hids

### ISSUE THREAD

**Issue by vaibhavsood** [ossec/ossec-hids#1269]
Hi,

Want to check if there is a docker image/Dockerfile for ossec-hids which is officially supported? I could not find one under dockerhub official images https://hub.docker.com/explore/

If not, want to check if there is any plan to add an official ossec-hids image to dockerhub? https://docs.docker.com/docker-hub/official_repos/#how-do-i-create-a-new-official-repository

**Comment by atomicturtle** [comment]
yes indeed there is, we're just wrapping up on the documentation side. If you want to get at the pre-release it is:

https://hub.docker.com/r/atomicorp/ossec-docker/

and the source repo will be in the ossec repo soon. You can see the work in progress here:

https://github.com/atomicturtle/ossec-docker

**Comment by vaibhavsood** [comment]
Thanks for the response!

My primary interest for now is to understand if it makes sense to have ossec-hids as a docker official image (steps for doing the same are in the link posted in my previous comment, it also mentions the considerations for making an image official)

If that happens, the image comes under Docker's "official" image program, they then take over the daily builds etc, also it gets published under https://hub.docker.com/explore/ - however the maintainence is to be done by upstream

So in summary wanted to check if doing this make sense or hosting the image under https://hub.docker.com/r/atomicorp/ossec-docker/ would be the only way for this docker image?

**Comment by atomicturtle** [comment]
Yes indeed, it will be duplicated as the official repo. https://hub.docker.com/r/atomicorp/* is tied into our automatic CI so builds will always end up there first
